test/integration/puppetlabs/puppetserver/testutils.clj

```diff
@@ -125,6 +125,31 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Interacting with puppet code and catalogs
 
+(schema/defn ^:always-validate
+  module-dir :- File
```

Comment:
Ah that was something I noticed was happening in `master-core`, so I copied that since I looked there last. Guess they're stylistically different across namespaces :)